TreeMaker/python/makeTreeFromMiniAOD_cff.py

```diff
@@ -264,8 +265,8 @@ def makeTreeFromMiniAOD(self,process):
         ak4updates = cms.PSet(discrs = cms.vstring())
         TMeras.TM80X.toModify(ak4updates,
             discrs = cms.vstring(
-                ['pfDeepCSVJetTags:'+x for x in ['probb','probc','probudsg','probbb']] +
-                ['pfDeepCSVDiscriminatorsJetTags:'+x for x in ['BvsAll','CvsB','CvsL']]
+                ['pfDeepCSVJetTags:'+x for x in ['probb','probbb']] +
```

Comment:
follow up if we actually need these (and not the others)

Reply:
These have also been removed.